less/_width.mq.less

```diff
@@ -0,0 +1,32 @@
+// SMALL
+@media @mq-sm, @mq-md {
+	.w {
+		 
+		@list: 20, 25, 33, 40, 50, 60, 66, 75, 80;
+		@listLength: length(@list);
+
+		.loop(@arrayLength, @index: 1) when (@index =< @arrayLength) {
+			@arrayVal: extract(@list, @index);
+			@pcnt: percentage(@arrayVal / 100);
+
+			> .w@{arrayVal} {
+				display: block;
```

Comment:
we don't have a loop for the page layout classes, but I am sure we could do some condensing with our page and widget layout styles